automation/Extension/webext-instrumentation/src/background/callstack-instrument.js

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
something is confusing - why does this live here as well as in the feature.js webpack dir? Aren't we doing some automated copying of files? @englehardt can you remind me how this works?

Reply:
Sorry, this is a leftover from my first attempt of implementing the listener. It doesn't belong here